ttl/hbp_ephys_stimuli.ttl

```diff
@@ -0,0 +1,161 @@
+@prefix : <http://www.hbp.FIXME.org/hbp_ephys_stimuli/> .
+@prefix nsu: <http://www.FIXME.org/nsupper#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix HBP_EPST: <http://www.hbp.FIXME.org/hbp_ephys_stimuli/> .
+
+<http://www.hbp.FIXME.org/hbp_ephys_stimuli> rdf:type owl:Ontology ;
+	owl:imports <https://raw.githubusercontent.com/NeuroscienceKnowledgeSpace/methodsOntology/master/ttl/nsupper.ttl> .
+# 
+# 
+# #################################################################
+# #
+# #    Classes
+# #
+# #################################################################
+# 
+# 
+# http://www.hbp.FIXME.org/hbp_ephys_stimuli/0000000
+
+HBP_EPST:0000000 rdf:type owl:Class ;
+	rdfs:label "Electrophysiological Stimuli" .
```

Comment:
Change HBP_EPST:0000000 label from  'Electrophysiological Stimuli' to 'Electrophysiological Stimuli Type'